common/oidc-authservice/base/statefulset.yaml

```diff
@@ -18,7 +18,7 @@ spec:
       serviceAccountName: authservice
       containers:
       - name: authservice
-        image: gcr.io/arrikto/kubeflow/oidc-authservice:e236439
+        image: docker.io/kubeflowmanifestswg/oidc-authservice:e236439
```

Comment:
does it need to be changed in /base/kustomization.yaml file?

Reply:
Good catch! Updated